dev-utils/integration-test.js

```diff
@@ -44,6 +44,11 @@ async function runIntegrationTest(pageUrl) {
     })
 
     await page.goto(pageUrl, { timeout: 30000 })
+    await page.evaluate(() => {
+      // From puppeteer 2.0 onwards the Chromium included with it supports the CompressionStream api.
+      // Because of that, the postData property is set to undefined causing the test to fail
+      CompressionStream = void 0
+    })
```

Comment:
@vigneshshanmugam, this one has been tricky b to find:

**Puppeteer request having Compression API enabled:**

`{ hasPostData: true, postData: undefined }`

**Puppeteer request having Compression API disabled:**

`{ hasPostData: true, postData: [...data...] }`

Reply:
Good catch, Yes that should explain the integration test failure on rum package.